syntax = "proto3";

message QueryRequest {
	string query = 1;
	string context = 2;
}

message QueryResponse {
	string response = 1;
}

service VectorDBLLM {
	rpc Query (QueryRequest) returns (QueryResponse);
}
